runtime/vm/vmprops.c

```diff
@@ -309,6 +316,8 @@ addPropertyForOptionWithModuleListArg(J9JavaVM *vm, const char *optionName, IDAT
 					}
 				}
 				CONSUME_ARG(j9vm_args, argIndex);
+
+				j9mem_free_memory(optionArg);
```

Comment:
Above there is a case that does `goto _end` which needs to free optionArg

Reply:
two cases actually

Reply:
Fixed the cases as mentioned above.